lib/web_ui/lib/src/engine/skwasm/skwasm_impl/raw/raw_memory.dart

```diff
@@ -85,6 +85,18 @@ class StackScope {
     return pointer;
   }
 
+  Pointer<Float> convertRectsToNative(List<ui.Rect> rects) {
+    final Pointer<Float> pointer = allocFloatArray(rects.length * 4);
+    for (int i = 0; i < rects.length; i++) {
+      final ui.Rect rect = rects[i];
+      pointer[i * 4] = rect.left;
+      pointer[i * 4 + 1] = rect.top;
+      pointer[i * 4 + 2] = rect.right;
+      pointer[i * 4 + 2] = rect.bottom;
```

Comment:
`+ 3`?

Reply:
Doh! I obviously need to write some unit tests here still.